pull_request_template.md

```diff
@@ -0,0 +1,41 @@
+# <pull_request_title>
+
+[Jira Ticket](<link_to_jira_ticket>)
+- [ ] Ready for review
+
+## Checklist
+- [ ] I have completed a self review
+- [ ] I have updated the [spreadsheet](https://docs.google.com/spreadsheets/d/1277nPOKtBFOk_kCo870_zA460-qlw_kGnYlLPwtnadE/edit#gid=0)
+- [ ] I have updated the jira ticket
+- [ ] I have added the relevant labels to this PR
+- [ ] I have updated documentation (if applicable)
+- [ ] I have run the unit tests suite and they pass
+- [ ] I have updated the [change log](https://docs.google.com/document/d/1t3s6lQuUcug1Fj8E_i2gBJNvXN0-funwBArG2hsia6k/edit)
+
+
+## Description
+
+## Notes
+
+## Dependencies for merge
+- [ ] [Pull request link]()
+
+## Testing
+
+### Visual Test
+- [ ] Not applicable
+- [ ] Own computer
```

Comment:
Is own computer really necessary in case of having c7 c10? If you're doing it on console, it would be a waste of time to do on pc. In case you're doing it on pc in both sizes, that's equal so

Reply:
Also it only applies to you guys on windows, us on Mac can't test locally

Reply:
The main reason why I added that is because sometimes for small features (at least me personally) I only test on my laptop and don't bother with the console. Or same if I don't have access to a console when I'm working

Reply:
Hmm okay, what I mean though is:
You need to test both sizes, if you're doing it on PC, you're testing both console sizes and if that meets the criteria for testing then it's equivalent to c7/c10.